backend/src/types.ts

```diff
@@ -0,0 +1,10 @@
+import { RowDataPacket } from "mysql2";
+
+export interface sticky_notes extends RowDataPacket {
```

Comment:
`interface`や`type`の時PascalCaseを使うことが多いイメージなのでこっちの方が良いかもです
```suggestion
export interface StickyNotes extends RowDataPacket {
```